### PR TITLE
Add files to build rpm

### PR DESCRIPTION
**Pull request by JanZerebecki**

### CODE REVIEW

File: suseconnect-ng.spec [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+%prep
+%setup -q -n connect-ng-%{version}
+
+%build
+find %_builddir/..
+%goprep %{import_path}
+find %_builddir/..
+go list -m all
+%gobuild cmd
+go build -buildmode=c-shared -o %_builddir/go/src/github.com/SUSE/connect-ng/ext/libsuseconnect.so ext/main.go
```
Comment (djoreilly) [inline suseconnect-ng.spec:50]: This extension library will not be needed for the time being. It's purpose in the PoC was to demonstrate how Ruby could call a library written in Go. It exposes no useful functions at the moment.

**Review by djoreilly — COMMENTED** [review]
Looks ok to me, but I have no experience building RPMs for Go programs

File: suseconnect-ng.spec (hunk 1 above)
Comment (JanZerebecki) [reply suseconnect-ng.spec:50]: Yes, the result is not used, I just build it for CI purposes.

**Review by djoreilly — APPROVED** [review]
lgtm.

**Review by skazi0 — APPROVED** [review]
lgtm, but I also haven't built any go packages so my experience is limited.

**Pull request merged**